src/features/changeForwarding.ts

```diff
@@ -69,6 +69,8 @@ function forwardFileChanges(server: OmniSharpServer): IDisposable {
                     // being that the file is now in an inconsistent state.
                     // If the document is closed, however, it will no longer be synchronized, so the text change will
                     // not be triggered and we should tell the server to reread from the disk.
+                    // This applies to C# code files only, not other files significant for OmniSharp
+                    // e.g. csproj or editorconfig files
```

Comment:
Do we not get incremental updates for these files as well that we then send to the server?

Reply:
at the moment there are quite a few things that need to fire as callbacks in response to file changed events coming via the files changed endpoint (https://github.com/OmniSharp/omnisharp-roslyn/blob/master/src/OmniSharp.MSBuild/ProjectManager.cs#L379-L477). 
At the moment none of them are arriving